packages/cli/src/tools/packageManager.js

```diff
@@ -36,33 +62,33 @@ export function setProjectDir(dir: string) {
 }
 
 export function install(packageNames: Array<string>, options?: Options) {
-  return shouldUseYarn(options)
-    ? executeCommand('yarn', ['add', ...packageNames], options)
-    : executeCommand(
-        'npm',
-        ['install', ...packageNames, '--save', '--save-exact'],
-        options,
-      );
+  configurePackageManager(
+    shouldUseYarn(options),
```

Comment:
I think it makes sense for `shouldUseYarn` to be moved to `configurePackageManager`. We already have `options`, so we could change the fn signature to be:

```js
function configurePackageManager(
  packageNames: Array<string>,
  options?: Options,
  action: 'add' | 'addDev' | 'remove' | 'install',
)
```